### ISSUE TITLE
Getting the Sensor Data multiple Times in the Serial

### ISSUE THREAD

**Issue by Caliban2017** [Zwer2k/WeatherStationDataRx#9]
Hello, first of all a big Thank You for your great Work, it is working very good so far.

I use the Ventus W132 connected to a WeMos D1 Mini with ESP8266. It shows the in the Serial all Data, but it will send them sometimes multiple Times. So one Data is incoming and this Text for Example Repeats 3 times in a short distance:

Temperature: 27.00°C
Humidity: 46%
Battery: OK
Temperature: 27.00°C
Humidity: 46%
Battery: OK
Temperature: 27.00°C
Humidity: 46%
Battery: OK

What is causing that? How can i limit it that the Incoming Data is showed only once? I did not modified anything so far. 

Thank you very much and greetings from Germany

**Comment by Caliban2017** [comment]
I have placed a Delay after the Serial printing of the Sensor-Data for about 2-3 Seconds and that help to fix the Problem with the multiple Output.

But after some testing the Script there are some critical Problems. I don't know what it causes but randomly the Wemos seems to be crashing. I see a big amount of Exception Messages. I tried to read the Data with the Exception Decoder Tool in Arduino IDE but i can not debug the Results, because my klowledge for that is not good enough. 

The Wemos resets after the Crash itself, but Wifi is not connecting anymore. Only after a RST Button press it is working again for some time until the next Crash. For the Moment i solved the Problem that i pull D7 Pin to Low connected to RST Pin, to restart the Wemos. After that it is working again but of course the Position of the Wind Direction is not correct anymore.

But sadly it seems like here is not happening anything anymore...?
